web/src/assets/i18n/de.json

```diff
@@ -225,6 +225,7 @@
 		"addStart": "Start",
 		"addStep": "Schritt",
 		"delete": "Löschen",
+		"layout": "Layout",
```

Comment:
Layout is maybe not the best word for this. Maybe _Auto-Layout_ or _Auto-Positioning_ or _Auto-Format_ or something like this